src/client/LangSelector.ts

```diff
@@ -2,27 +2,28 @@ import { LitElement, html } from "lit";
 import { customElement, state } from "lit/decorators.js";
 import "./LanguageModal";
 
-import ar from "../../resources/lang/ar.json";
-import bg from "../../resources/lang/bg.json";
-import bn from "../../resources/lang/bn.json";
-import cs from "../../resources/lang/cs.json";
-import de from "../../resources/lang/de.json";
+import ar from "../../resources/lang/ar_SA.json";
+import bg from "../../resources/lang/bg_BG.json";
+import bn from "../../resources/lang/bn_BD.json";
+import cs from "../../resources/lang/cs_CZ.json";
+import de from "../../resources/lang/de_DE.json";
 import en from "../../resources/lang/en.json";
```

Comment:
Please explain the reasoning for lines 10 and 11.
Why have `en.json` and `en_US.json` both existing and imported?